### DISCUSSION TITLE
Does NeoPixelBus library support DotStarSpiMethod on ESP8266?

### DISCUSSION THREAD

**Discussion by wangnick** [Makuna/NeoPixelBus#631] [Q&A] [answered]
Hi, quick question: Does NeoPixelBus library support DotStarSpiMethod on ESP8266? I tried to find the answer in the documentation, but it speaks mostly of the ESP32 in that regards ...

**Comment by wangnick** [comment, marked as answer]
And the answer is: Yes, it does.

**Comment by Makuna** [comment]
DotStars are more flexible on timing since they rely on a clock signal, so the Arduino SPI even in software works across all platforms.
